suite-native/receive/src/hooks/useAccountReceiveAddress.tsx

```diff
@@ -121,9 +143,9 @@ export const useAccountReceiveAddress = (accountKey: AccountKey) => {
         }
 
         return false;
-    }, [accountKey, freshAddress, dispatch, showToast, navigation, showAlert]);
+    };
 
-    const handleShowAddress = useCallback(async () => {
+    const handleShowAddress = async () => {
```

Comment:
Why was this `useCallback` removed? Since the function is returned from this hook, every run of it will cause a rerender in any component/hook that uses `handleShowAddress`